src/libraries/System.Private.Xml/tests/XmlWriter/DisposeTests.cs

```diff
@@ -121,5 +123,161 @@ public static void XmlWriterDisposeWorksWithDerivingClasses()
             mywriter.Dispose();
             Assert.True(mywriter.IsDisposed);
         }
+
+        [Fact]
+        public static async Task AsyncWriter_DisposeAsync_ShouldCall_FlushAsyncWriteAsyncOnly_StreamWriter()
+        {
+            using (var stream = new AsyncOnlyStream())
+            await using (var writer = XmlWriter.Create(stream, new XmlWriterSettings() { Async = true }))
```

Comment:
should this ConfigureAwait?

Reply:
Feedback got from @stephentoub `The .ConfigureAwait(false) usage can/should be removed in the tests.` https://github.com/dotnet/runtime/pull/35233#discussion_r412256260